packages/react/src/analytics/integrations/shared/GoogleConsentMode/types.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * Interface defining the structure of consent category mappings.
+ */
+
+export interface GoogleConsentOptions {
+  ad_storage: Array<string>;
+  ad_user_data: Array<string>;
+  ad_personalization: Array<string>;
+  analytics_storage: Array<string>;
+}
+
+export interface GoogleConsentMappings {
```

Comment:
This is the same as the previous type

Reply:
Also, this type does not need to be exported and I would give it another name that is more distinguishable like `GoogleConsentMappingsBase`

Reply:
Fixed